ZelBack/src/services/benchmarkService.js

```diff
@@ -249,7 +249,7 @@ async function executeUpnpBench() {
   const isUPNP = upnpService.isUPNP();
   if ((userconfig.initial.apiport && userconfig.initial.apiport !== config.server.apiport) || isUPNP) {
     log.info('Calling FluxBench startMultiPortBench');
-    startMultiPortBench();
+    log.info(await startMultiPortBench());
```

Comment:
That is pretty strange @Cabecinha84

Reply:
what you get:
2023-04-20T16:04:05.740Z          Calling FluxBench startMultiPortBench
2023-04-20T16:04:05.742Z          {"status":"success","data":"Multiport Benchmarks will start in the next couple minutes"}